### PR TITLE
[API] Refactor code + rewrite training code in python 

### PR DESCRIPTION
**Pull request by yashhshah**
### Summary
This PR accomplishes the following: 
1. change all python code to adhere to pylint standards
2. refactor methods to be used by training code and server code into helpers.py
3. rewrite training code from .js to .py 
4. add more robustness to db item query by removing the item if a field we want does not exist (ensures robustness of structure of data to a certain extent) 

### Test Plan
1. start server
2. ensure curls work and a response is returned 
3. run python training script and ensure model is updated

### CODE REVIEW

File: python/server.py [hunk 1]
```diff
@@ -197,123 +67,18 @@ def classify_one(id, uri=None):
...
-  else:
-    tokens.append("not_containsEmoji")
-
+  tokens = extract_tokens(token_id, rpcUrl)
   classification = classify(tokens)
 
   cnftTable.put_item(
     Item={
-      'address': id,
+      'address': token_id,
```
Comment (yashhshah) [inline python/server.py:84]: changing the id reserved keyword to token_id

File: python/helpers.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    """
+    response = imageTable.get_item(
+        Key={
+            'url': image_url,
+        }
+    )
+    if "Item" in response:
+        if "words" in response["Item"].keys():
+            return response["Item"]["words"]
+        # A malformed entry has made its way into the table
```
Comment (yashhshah) [inline python/helpers.py:76]: attempt at removing malformed entries since this is kind of a required field

Comment (solarnius) [reply python/helpers.py:76]: do we have malformed entries rn?

Comment (yashhshah) [reply python/helpers.py:76]: as part of running the train script I think I created them for a little bit but we shouldnt now that Ive run them again

File: python/server.py (hunk 1 above)
Comment (solarnius) [reply python/server.py:84]: so that's what the special syntax highlighting meant

File: python/helpers.py (hunk 2 above)
Comment (solarnius) [reply python/helpers.py:76]: makes sense, we should always be pushing to `words` even if it's an empty array i believe so this should be redundant but doesn't hurt

File: python/train.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        print(f"trained {token_id} as spam {i + 1}/{len(spam_ids)}")
+
+    for i, token_id in enumerate(ham_ids):
+        tokens = extract_tokens(token_id, RPC_URL)
+        train("ham", tokens)
+        print(f"trained {token_id} as ham {i + 1}/{len(ham_ids)}")
+
+def clean_model():
+    """
+    Remove token lengths that can distract model results
```
Comment (solarnius) [inline python/train.py:52]: i think we want to remove short tokens as well here

we're removing infrequent ones with the `model[category]["tokens"][token]` check but we should do a `len` check as well to get rid of noisy single letters

Comment (yashhshah) [reply python/train.py:52]: added

**Pull request merged**